docs/quick-start/quickstart-users.md

```diff
@@ -50,13 +50,10 @@ the Project Code:
     3.  Select the correct project from the drop down list
     4.  Select the *archer2* machine in the list of available machines
     5.  Click *Next*
-    6.  Enter a username for the account and (optionally) an SSH public
-        key
-        1.  If you do not specify an SSH key at this stage, your default
-            key will be used (if you have one). For users who had an ARCHER
+    6.  Enter a username for the account and an SSH public key (optional; if you already uploaded a default SSH key to your SAFE account, this will be used)
+        1.  If you do not specify an SSH key at this stage, your default key will be used. For users who had an ARCHER
```

Comment:
I am not sure I would bother to clarify and try to distinguish the optional nature. Just assume people have to add it here. How about:

6. Enter a username for the account and the public part of an SSH key pair
    1. More information on generating SSH key pair [can be found in the ARCHER2 User and Best Practice Guide](https://docs.archer2.ac.uk/user-guide/connecting](https://docs.archer2.ac.uk/user-guide/connecting/#ssh-key-pairs).
    2. You can add additional SSH keys using the process described below if you so wish.

Reply:
Also, get rid of the mention of ARCHER accounts now - it has been long enough